### PR TITLE
Adding Developer, branding guidelines. Updates to rendition flow.

### PR DESCRIPTION
**Pull request by slohia-adobe**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (mpeercy) [inline .gitignore]: Do we need this addition to gitignore?

File: manifest.json [hunk 2]
```diff
@@ -50,7 +50,17 @@
...
+					"title": "Read and Generate Renditions"
+				},
+				{
+					"pages": [],
+					"path": "quickstart/developer.md",
+					"title": "Developer Guildelines"
+				},
+				{
+					"pages": [],
+					"path": "quickstart/renditions.md",
```
Comment (mpeercy) [inline manifest.json]: This should be "branding.md".

File: quickstart/oauth.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ The OAuth workflow is described in detail in the [Adobe IMS Authentication and A
 
 Partner applications must include the `lr_partner_apis` scope (along with the standard `openid` and `AdobeID` scopes) to access the Lightroom Services.
 
+To allow renew of refresh token, additional setup is needed on the API key. Should your application require this capability, you must contact Adobe (need info from Drew) and describe your needs.
```
Comment (mpeercy) [inline quickstart/oauth.md]: Should remove "(need info from Drew)".

File: quickstart/renditions.md [hunk 4]
```diff
@@ -1,59 +1,33 @@
...
-PUT /v2/catalogs/{catalog_id}/assets/{asset_id}/master HTTP/1.1
-Authorization: {auth_token}
-Content-Length: {xsd:nonNegativeInteger}
-Content-Range: {xsd:string}
-Content-Type: {xsd:string}
-```
+##Workflow to generate renditions for print workflow
 
-Sample success response:
+Generate renditions for an original file asynchronously. Allowed rendition types are fullsize and 2560. Generated rendition will be deleted after 1 day automatically. Both these renditions must always be requested on demand.  First check to see if one is available, and if not create it. These renditions are not available with basic API access. Should your application require this capability, you must contact Adobe (lightroomcontact@adobe.com) and describe your needs. Here are the details of fullsize and 2560
```
Comment (mpeercy) [inline quickstart/renditions.md]: Should remove "(lightroomcontact-at-adobe.com)".

**Review by mpeercy — CHANGES_REQUESTED** [review]
As per the comments, there are a handful of changes that should be made before merging.

File: quickstart/renditions.md (hunk 4 above)
Comment (slohia-adobe) [reply quickstart/renditions.md]: @mpeercy Is it fine to include to just contact Adobe and not mention any email address for now? Folks need to raise issue in order to contact us then.

File: manifest.json (hunk 2 above)
Comment (slohia-adobe) [reply manifest.json]: yeah will fix it.

**Review by mpeercy — APPROVED** [review]
LGTM.

**Pull request merged**